app/serializers/api/v3/top_profiles_serializer.rb

```diff
@@ -0,0 +1,32 @@
+module Api
+  module V3
+    class TopProfilesSerializer < ActiveModel::Serializer
```

Comment:
@Bluesmile82 or @sorodrigo can you take a look at this serializer and let Marta know in case you need any more data for the top profiles feed? Also if you need any filters on this endpoint.

Reply:
👍